framework/src/wifi_manager/wifi_manager.c

```diff
@@ -541,8 +541,10 @@ _convert_scan_info(wifi_manager_scan_info_s **wm_scan_list, wifi_utils_scan_list
 		cur->rssi = iter->ap_info.rssi;
 		cur->channel = iter->ap_info.channel;
 		cur->phy_mode = iter->ap_info.phy_mode;
-		strncpy(cur->ssid, (char *)iter->ap_info.ssid, 32);
-		strncpy(cur->bssid, (char *)iter->ap_info.bssid, 17);
+		cur->ap_auth_type = iter->ap_info.ap_auth_type;
+		cur->ap_crypto_type = iter->ap_info.ap_crypto_type;
+		strncpy(cur->ssid, (char *)iter->ap_info.ssid, WIFIMGR_SSID_LEN);
```

Comment:
Please refer to the comment that pillip8282 asked yesterday (see outdated comment above). Briefly, because it uses calloc(), we do not need to insert null at the end of the string. Thanks for the comment.

Reply:
(Additional comment) as far as I know, WiFi standard allows SSID length up to 32 bytes. In general, 32nd byte is filled with null, however, it can be a meaningful character depending on the implementation (we have to verify this sentence is true, based on the standard). In that case, we have to copy 32 bytes not to cut the last character, and put null at the end of the string. We will cover this issue at the next PR, after confirming some facts. Thanks.

Reply:
I don't understand what relationship between calloc and length of string copy. As @JeongJunSik said, why not ```WIFIMGR_SSID_LEN - 1```?
Regardless of this chage, why do you use calloc instead of zalloc even though it has 1 as n arg?

Reply:
```In that case, we have to copy 32 bytes not to cut the last character, and put null at the end of the string. We will cover this issue at the next PR, ```
Do you mean increasing buffer size to 33?

Reply:
Hi, @sunghan-chang and @JeongJunSik. That is true. In the next PR (I am woring on now), we will change the size of SSID and passphrase string to 33 and 65 bytes, respectively. Then, copying of 32 and 64 bytes will make sense. These change will be applied entire system. Sorry for the confusion.